CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+## [22.0.0] - 2025-02-24
+
+-   **[Breaking] Makes URL path normalization case sensitive**
```

Comment:
Please use the format we use in older versions

Reply:
Updated in f66c1cb